src/site/xdoc/cmdline.xml.vm

```diff
@@ -450,6 +450,355 @@ java -D&lt;property&gt;=&lt;value&gt;  \
       </div>
     </section>
 
+        <section name="Examples">
+        <h3>1. Using a Configuration File (-c)</h3>
```

Comment:
Unfortunately `-c` rendering as `-C` can we update css to avoid this for this special headers ?
If not, let's remove names of parameters.